packages/notifi-react-card/lib/components/intercom/MessageGroup.tsx

```diff
@@ -43,12 +39,13 @@ export const MessageGroup: React.FC<MessageGroupProps> = ({
               : 'NotifiIntercomChatOutgoingMessage__container',
             classNames?.messageContainer,
           )}
+          key={index}
         >
-          {isIncoming ? (
+          {/* {isIncoming ? (
```

Comment:
Remove comment

Reply:
Oh, this is the sender's avatar that we may need to render when we have the correct data field to exposed, so comment this for now, mentioned in the description box.

Reply:
check in the client if there is another api that we need to pull that includes the user's avatar